aggregator/src/aggregator/http_handlers.rs

```diff
@@ -141,8 +142,23 @@ async fn run_error_handler(error: &Error, mut conn: Conn) -> Conn {
                 .with_task_id(&inner.task_id)
                 .with_detail(&inner.to_string()),
         ),
+        Error::Datastore(
+            error @ datastoreError::TimeUnaligned {
+                task_id,
+                time_precision: _,
+                inner_error: _,
```

Comment:
nit: to more compactly elide the rest of the struct, use `..` as in `datastoreError::TimeUnaligned { task_id, .. }`